src/shared/types/targeting/gutter.ts

```diff
@@ -0,0 +1,15 @@
+import { PageTracking, Tag } from './shared';
+
+export interface GutterTargeting {
+    showSupportMessaging: boolean;
+    countryCode: string;
+    mvtId: number;
+    isSignedIn: boolean;
+    tags: Tag[];
```

Comment:
I know we use this model for epics, but I think `tagIds` would also be enough as it's what we use for the banner -
https://github.com/guardian/support-dotcom-components/blob/main/src/shared/types/targeting/banner.ts#L17

Reply:
Ok, thanks Tom - I've changed that now.